### ISSUE TITLE
Inquiry regarding Training Efficiency and PPO Convergence Consistency

### ISSUE THREAD

**Issue by YellowThree-HS** [xcwoid/BVRGym#6]
First of all, thank you for open-sourcing this environment. It is a very valuable contribution to the field of high-fidelity BVR air combat simulation.

I am currently trying to reproduce the results presented in your paper, but I have encountered two major issues that I hope you could help clarify:

1. Training Efficiency Discrepancy The paper mentions that "One day's worth of computing is needed to solve the problem using... ten CPUs operating in parallel". However, in my experiments using mainBVRGym_MultiCore.py with similar multi-core resources (8-10 cores), the training speed is significantly slower than expected. Currently, I am only able to run approximately 4,000 to 10,000 episodes per day. At this rate, reproducing the 100k+ episodes required for convergence would take weeks.

Question: Could you verify if there are specific simulation step settings (e.g., r_step in the config) that were used in the paper to accelerate the integration? The default configuration seems to run the JSBSim integration too frequently relative to the decision frequency.

2. Convergence with Default PPO Parameters Using the default PPO hyperparameters provided in the repository, I am unable to achieve stable convergence.

Observation: The reward curves tend to fluctuate heavily or plateau without reaching the performance levels shown in the paper's figures. The agent often learns to "drag" the missile until timeout rather than successfully evading or engaging.

Question: Are there updated hyperparameters or a specific random seed that you recommend for stable training?

Is there a newer version of the code available, or could you provide some insights into how to bridge the gap between the current repo performance and the results reported in the paper?

Thank you very much for your time and help.

<img width="1694" height="550" alt="Image" src="https://github.com/user-attachments/assets/162616de-8719-414f-b36b-4e39c1b43d15" />

<img width="2026" height="609" alt="Image" src="https://github.com/user-attachments/assets/66d1c026-a5e0-4b11-8045-669abe839736" />

**Comment by xcwoid** [comment]
I will restructure the repo a bit, and then verify
